common/src/main/java/io/netty/util/Recycler.java

```diff
@@ -349,6 +358,20 @@ boolean transfer(Stack<?> dst) {
                 return false;
             }
         }
+
+        @Override
+        protected void finalize() throws Throwable {
+            super.finalize();
```

Comment:
super.finalize() should never throw. But sure why not.. Everything else ok?

> Am 24.07.2016 um 14:55 schrieb Scott Mitchell <notifications@github.com>:
> 
> In common/src/main/java/io/netty/util/Recycler.java:
> 
> > @@ -349,6 +358,20 @@ boolean transfer(Stack<?> dst) {
> >                  return false;
> >              }
> >          }
> > +
> > +        @Override
> > +        protected void finalize() throws Throwable {
> > +            super.finalize();
> use a try/finally to ensure the reclaim always happens?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>